packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 All notable changes to the "zowe-explorer-api" extension will be documented in this file.
 
-## `2.4.1`
+## TBD Release
+
+- New API call `getJobsByParameters` to allow filtering jobs by status.
```

Comment:
Could we add this new feature under the `New features and enhancements`?
Similar to the [v2.3.0 changelog](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer/CHANGELOG.md#new-features-and-enhancements-2)